pom.xml

```diff
@@ -254,10 +253,27 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <!-- inheriting general ossrh-release setup from aissemble-parent -->
+            <id>ossrh-release</id>
+            <properties>
+                <maven.build.cache.enabled>false</maven.build.cache.enabled>
```

Comment:
I: Ensure the build cache isn't used during the release plugins release goals. This was easier than sprinkling the argument in different configs for the release plugin.